app/components/listing/index.tsx

```diff
@@ -6,7 +6,6 @@ export * from "./OrganizationCard";
 export * from "./OrganizationDetails";
 export * from "./RelativeOpeningTime";
 export * from "../ui/Cards/ServiceCard";
-export * from "./ServiceDetails";
```

Comment:
@adriencyberspace you also questioned this was used anywhere. I grep'd the code, it's nowhere to be found. ✂️